src/AutoComplete/AutoComplete.js

```diff
@@ -286,7 +293,9 @@ class AutoComplete extends Component {
   };
 
   chosenRequestText = (chosenRequest) => {
-    if (typeof chosenRequest === 'string') {
+    if (typeof this.props.chosenRequestText === 'function') {
+      return this.props.chosenRequestText(chosenRequest)
```

Comment:
Add a semicolon ( ; ) after `return this.props.chosenRequestText(chosenRequest)`